backend/spec/fixtures/snapshots/api/v1/enums.json

```diff
@@ -334,216 +334,230 @@
     },
     {
       "id": "academic",
+      "type": "location_type",
+      "attributes": {
+        "name": "Basins Category"
+      }
+    },
+    {
+      "id": "business-cooperative",
+      "type": "location_type",
+      "attributes": {
+        "name": "Basin"
+      }
+    },
+    {
+      "id": "business-large-enterprise-corporation",
       "type": "project_developer_type",
       "attributes": {
         "name": "Academic"
       }
     },
     {
-      "id": "business-cooperative",
+      "id": "business-sme-and-msme",
       "type": "project_developer_type",
       "attributes": {
         "name": "Business - Cooperative"
       }
     },
     {
-      "id": "business-large-enterprise-corporation",
+      "id": "business-start-up-entrepreneur",
       "type": "project_developer_type",
       "attributes": {
         "name": "Business - Large enterprise corporation"
       }
     },
     {
-      "id": "business-sme-and-msme",
+      "id": "business-trade-or-business-association",
       "type": "project_developer_type",
       "attributes": {
         "name": "Business - SME and MSME"
       }
     },
     {
-      "id": "business-start-up-entrepreneur",
+      "id": "government-national",
       "type": "project_developer_type",
       "attributes": {
         "name": "Business - Start-up entrepreneur"
       }
     },
     {
-      "id": "business-trade-or-business-association",
+      "id": "government-sub-national",
       "type": "project_developer_type",
       "attributes": {
         "name": "Business - Trade or business association"
       }
     },
     {
-      "id": "government-national",
+      "id": "public-private-organization-or-program",
       "type": "project_developer_type",
       "attributes": {
         "name": "Government - National"
       }
     },
     {
-      "id": "government-sub-national",
+      "id": "ngo",
       "type": "project_developer_type",
       "attributes": {
         "name": "Government - Sub-national"
       }
     },
     {
-      "id": "public-private-organization-or-program",
+      "id": "iplc-organization",
       "type": "project_developer_type",
       "attributes": {
         "name": "Public-private organization or program"
       }
     },
     {
-      "id": "ngo",
+      "id": "incipient",
       "type": "project_developer_type",
       "attributes": {
         "name": "NGO"
       }
     },
     {
-      "id": "iplc-organization",
+      "id": "consolidaton",
```

Comment:
This snapshot is totally weird.

Reply:
maybe, this is because `id` is treated as dynamic attribute globally in `rails_helper`. And for some endpoints it is not dynamic, as this one for example. Explicit reset dynamic_attributes in snaphost matches could resolve this, but I'm not sure.